### PR TITLE
Update cli to 2.0.1-beta.34

### PR DESCRIPTION
**Pull request by pragnagopa**

### CODE REVIEW

File: cli-feed-v3.json [hunk 1]
```diff
@@ -169,6 +169,19 @@
       "FUNCTIONS_EXTENSION_VERSION": "beta",
       "requiredRuntime": ".NET Core",
       "minimumRuntimeVersion": "2.1"
+    },
+	 "2.3.3": {
+      "Microsoft.NET.Sdk.Functions": "1.0.14",
+      "cli": "https://functionscdn.azureedge.net/public/2.0.1-beta.34/Azure.Functions.Cli.win-x86.2.0.1-beta.34.zip",
+      "nodeVersion": "8.11.1",
+      "localEntryPoint": "func.exe",
+      "itemTemplates": "https://www.myget.org/F/azure-appservice/api/v2/package/Azure.Functions.Templates/2.0.0-beta-10210",
```
Comment (pragnagopa) [inline cli-feed-v3.json]: @soninaren - Any updates to  itemTemplates, projectTemplates or templateApiZip? I copied these from 2.3.2

**Comment by fabiocav** [comment]
For  OGF testing, we need to make sure the actual release is not updated, otherwise customers will get this.

File: cli-feed-v3.json (hunk 1 above)
Comment (soninaren) [reply cli-feed-v3.json]: Yes there are some changes, let me send you the new links

Comment (soninaren) [reply cli-feed-v3.json]: Sha is incorrect
4A2B808E86AE4C4DEF38A2A14270D19EC384648AD1FDF635921A64F609D41098

Comment (ahmelsayed) [reply cli-feed-v3.json]: btw, I updated the build script to generate sha2 files for all the different packages.

**Pull request merged**